metadata_service/proxy/atlas_proxy.py

```diff
@@ -416,9 +459,41 @@ def get_tags(self) -> List:
                 )
         return tags
 
-    def get_table_by_user_relation(self, *, user_email: str,
-                                   relation_type: UserResourceRel) -> Dict[str, Any]:
-        pass
+    def get_table_by_user_relation(self, *, user_email: str, relation_type: UserResourceRel) -> Dict[str, Any]:
+        params = {
+            'typeName': 'Reader',
+            'offset': '0',
+            'limit': '50',
+            'entityFilters': {
+                'condition': 'AND',
+                'criterion': [
+                    {
+                        'attributeName': 'qualifiedName',
```

Comment:
We do have the variable available for qualified name. Please use that instead `QN_KEY`

Reply:
Yes, I noticed it and have changed it. I hope we would eventually move them from this class as attributes grow.